R/decor.R

```diff
@@ -22,11 +22,15 @@ cpp_files <- function(pkg = ".") {
   }
 
   src <- file.path(pkg, "src")
-  if (dir.exists(src)) {
-    return(list.files(src, full.names = TRUE, pattern = "[.](cc|cpp|h|hpp)$"))
+  if (!dir.exists(src)) {
+    return(character())
   }
 
-  return(character())
+  out <- list.files(src, full.names = TRUE, pattern = "[.](cc|cpp|h|hpp)$")
```

Comment:
Is this something `fs::` could help with ? 

What's preventing to use `vec_sort()` ? cc @DavisVaughan @lionel- 

`decor` is mostly (only?) used at development time, so we can be easily depend on recent `vctrs`.

Reply:
`vec_order()` and `vec_sort()` don't use the C locale yet but will in a future vctrs release

Reply:
Thanks. We can revisit at this point then.